src/main/java/com/dmytrohuk/weborganizer/users/User.java

```diff
@@ -28,16 +21,24 @@ public class User {
     private String firstName;
 
     @Column(name = "last_name")
-    private String lastName;
+    private String surname;
```

Comment:
You are having unnecessary inconsistency between entity field name and database table column. Whether revert entity field name back to `lastName` or rename column to `surname`